R/bainCommon.R

```diff
@@ -782,6 +782,9 @@
     row <- list(hypotheses = gettext("Hu"), BFu = "", BF = "", PMP1 = "", PMP2 = bainResult[["fit"]]$PMPb[length(bainResult[["fit"]]$BF)])
     table$addRows(row) 
   }
+  
+  if(any(is.nan(unlist(bainResult[["fit"]]))))
+    table$addFootnote(message = gettext("<b>Warning:</b> The entered model contraints are incompatible with the data and therefore the computed results contain NaN's."))
```

Comment:
```suggestion
    table$addFootnote(symbol = gettext("<b>Warning</b>"), message = gettext("The entered model contraints are incompatible with the data and therefore the computed results contain NaN's."))
```
Otherwise you get `Note. Warning:`